aviatrix/data_source_aviatrix_dcf_webgroups.go

```diff
@@ -0,0 +1,103 @@
+package aviatrix
+
+import (
+	"context"
+
+	"github.com/AviatrixSystems/terraform-provider-aviatrix/v3/goaviatrix"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAviatrixDcfWebgroups() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceAviatrixDcfWebgroupsRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of the Web Group.",
+			},
+			"selector": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"match_expressions": {
```

Comment:
Does this work ok as a type list?  If you re-apply after applying do you see any ordering issues where terraform  "thinks" there is a change  do to the items being reordered?  This would be the controller for instance returning in a different order. If so this probably should be a set, or you need to sort below.